src/AccessLogEntry.cc

```diff
@@ -131,3 +129,10 @@ AccessLogEntry::effectiveVirginUrl() const
     return nullptr;
 }
 
+void
+AccessLogEntry::replyHeaders(MemBuf &mb) const
```

Comment:
I think we should change the name because we changed the API to return void.

```suggestion
AccessLogEntry::packReplyHeaders(MemBuf &mb) const
```

Reply:
Done.